linker/Linker.Steps/MarkStep.cs

```diff
@@ -30,11 +30,16 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Reflection;
```

Comment:
What do you need this for? This adds hard version dependency (e.g for type casts) which is very problematic for general IL tool.

Reply:
This is used for the `BindingFlags` enum. I can instead fully qualify `BindingFlags` where it is used. Is that preferred?

Reply:
My preference (if you really need it) is to use `SR` alias for System.Reflection

Reply:
Thanks. I guess this comes down to the use of `BindingsFlags` then. If we need to keep using it then, I'll switch this to `SR`. Otherwise, I'll remove it.